### ISSUE TITLE
Jetpack Connect: Links in error messaging don't pass color a11y

### ISSUE THREAD

**Issue by crunnells** [Automattic/wp-calypso#39782]
An issue surfaced in #39779 where link colors in the error message isn't fully accessible because of it's colors. The error text color is red, and the link is green with no underline, so it would not be distinguishable as a link for certain kinds of colorblindness. Screenshot:

https://user-images.githubusercontent.com/51896/75588202-fdb85b80-5a2c-11ea-99c2-3c700d7050dc.png

Here's an example of what it looks like with Green-Blind/Deuteranopia (generated via [Coblis](https://www.color-blindness.com/coblis-color-blindness-simulator/)).

![image](https://user-images.githubusercontent.com/4297811/75593642-14a97e80-5a2a-11ea-8fe8-f6e307ac68a2.png)

#### Steps to reproduce
1. Starting at URL: https://wordpress/log-in/jetpack (pending merge/release of #39779)
2. Use an email address that is not associated with a Jetpack account and click continue
3. The error message generated includes a link to create a new account.

Possible solutions: add an underline, or change the color value of the error link to something with a darker red. We're currently using [Color Studio](https://color-studio.blog/) Red 50, but the link could be Red 70 or 80.
